lib/milestone-db/no-op.js

```diff
@@ -0,0 +1,23 @@
+var MilestoneDB = require('./index');
+
+/**
+ * A no-op implementation of the MilestoneDB class.
+ *
+ * This class exists as a simple, silent default drop-in for ShareDB, which allows the backend to call its methods with
+ * no effect.
+ */
+module.exports = NoOpMilestoneDB;
+function NoOpMilestoneDB(options) {
+  MilestoneDB.call(this, options);
+}
+
+NoOpMilestoneDB.prototype = Object.create(MilestoneDB.prototype);
+
+NoOpMilestoneDB.prototype.getMilestoneSnapshot = function (collection, id, version, callback) {
+  process.nextTick(callback, null, undefined);
```

Comment:
I've noticed some other places in the code defining variables purely for argument clarity, might be worth doing here too - it took me a second to see why this was passing in `null` and then `undefined` as args.
```
var snapshot = undefined;
process.nextTick(callback, null, snapshot);
```